app/jobs/base_job.rb

```diff
@@ -52,39 +61,4 @@ def before(job)
       WorkerStatus.new(job).conditional_exit!
     end
   end
-
-  private
-
-  def running_in_correct_path?
-    return true unless File.exist?('config/exception_notifier.yml')
-
-    expected_path == STARTING_PATH
-  end
-
-  def notify_on_restart(msg)
-    Rails.logger.info msg
-    return unless File.exist?('config/exception_notifier.yml')
-
-    setup_notifier('DelayedJobRestarter')
-    @notifier.ping(msg) if @send_notifications
-  end
-
-  def expected_path
-    Rails.cache.fetch('deploy-dir') do
-      # A default for the first deployment and local development
-      # This should be set on deployment.
-      Delayed::Worker::Deployment.deployed_to
-    end
-  end
-
-  def unlock_job!(job_id)
-    a_t = Delayed::Job.arel_table
-    job_object = Delayed::Job.where(a_t[:handler].matches("%job_id: #{job_id}%").or(a_t[:id].eq(job_id))).first
-    return unless job_object
-
-    msg = "Restarting stale delayed job: #{job_object.locked_by}"
-    notify_on_restart(msg)
-
-    job_object.update(locked_by: nil, locked_at: nil)
```

Comment:
Is this all code related to the bare-metal deployments?  It looks like we moved this logic to the `WorkerStatus` class for use with ECS, just want to confirm.

Reply:
I believe this was all related to bare-metal deployments. Are you agreeing that it's dead code? That's what I was thinking it was at least.

Reply:
Yes, I was agreeing with that assessment, just wanted to make sure that's why you were removing it.